sct.py

```diff
@@ -1315,6 +1315,7 @@ def create_test_release_jobs(branch, username, password, sct_branch, sct_repo):
         ('repair-based-operation', "SCT RBO Tests"),
         ('scale', 'SCT Scale Tests'),
         ('operator', 'SCT Operator Tests'),
+        ('raft', 'SCT Raft Experimental'),
```

Comment:
this is build for prefix of the files, not the folder.

maybe `raft/` would work, but this would need to be cleaned from the folder name.

Reply:
added the missing part below